ui/src/app/tasks-jobs/schedules/schedule/schedule.component.html

```diff
@@ -1,34 +1,43 @@
 <h1>
-  Schedule <strong>{{ schedule?.name }}</strong>
+  {{ 'schedules.schedule.title' | translate }} <strong>{{ schedule?.name }}</strong>
 </h1>
 
 <div class="datagrid-action-bar">
-  <button type="button" class="btn btn-sm btn-secondary" (click)="detailsTask()">Task details</button>
-  <button type="button" class="btn btn-sm btn-outline-danger" (click)="destroy()">Delete schedule</button>
+  <button type="button" class="btn btn-sm btn-secondary" (click)="detailsTask()">
+    {{ 'schedules.schedule.taskDetails' | translate }}
+  </button>
+  <button type="button" class="btn btn-sm btn-outline-danger" (click)="destroy()">
+    {{ 'schedules.schedule.deleteSchedule' | translate }}
+  </button>
 </div>
 
 <div *ngIf="!isLoading">
   <div class="clr-row">
     <div class="clr-col-lg-6 clr-col-md-12 clr-col-12">
-      <app-view-card titleModal="Information" keyContext="schedule" name="{{ schedule.name }}" id="info">
+      <app-view-card
+        [titleModal]="'schedules.schedule.information' | translate"
+        keyContext="schedule"
+        name="{{ schedule.name }}"
+        id="info"
+      >
         <ng-template>
           <div class="block card-block-keyvalue">
             <div class="row">
-              <div class="key">Name</div>
+              <div class="key">{{ 'schedules.main.name' | translate }}</div>
```

Comment:
Texts like `Name`, `Platform`, `Status` and `Definition` are visibile on other pages.
Can we move these to `commons` translations ?